src/pykiso/lib/auxiliaries/communication_auxiliary.py

```diff
@@ -50,18 +50,28 @@ def send_message(self, raw_msg: bytes) -> bool:
         return self.run_command("send", raw_msg, timeout_in_s=None)
 
     def receive_message(
-        self, blocking: bool = True, timeout_in_s: float = None
+        self,
+        blocking: bool = True,
+        timeout_in_s: float = None,
+        clear_old: bool = True,
```

Comment:
clear_old ?? old what?

Reply:
I was going for empty_buffer but, Sebastian told me that